man/has_name.Rd

```diff
@@ -2,15 +2,7 @@
 % Please edit documentation in R/has-name.R
 \name{has_name}
 \alias{has_name}
-\title{Convenience function to check presence of a named element}
-\usage{
```

Comment:
`@inheritParams` didn't work, so I enclosed it in a wrapper. I opened an issue for inheriting examples: https://github.com/klutometis/roxygen/issues/588